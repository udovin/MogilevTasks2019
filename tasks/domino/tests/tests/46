746
14255567 277361
-18635638 33166
-94224956 440295
-3159933 362824
-5622686 773365
-83840587 671953
-23427445 413625
-11576922 408133
-24696954 336641
-59599614 773861
24393857 674610
19003623 757486
-39514307 616809
-87071849 647999
-40227144 779526
52687544 582816
-53149795 78489
49030033 108019
87374916 234492
84399893 515345
-27876400 458083
-22357096 344451
-43944999 96524
80054952 650274
-79201715 416442
3114671 185109
-96792913 648108
2081954 108853
-75538075 251266
732682 632425
6623055 504740
60310639 272348
82701124 493410
30695300 8395
-32423956 543896
8041567 112110
72336788 867242
11276687 707362
-76921497 274691
-49223252 224915
-76846817 907456
1455775 313476
-21259864 100118
39486702 123547
-23432224 953534
80787746 686089
-49679201 507451
95232310 858190
-38122947 394834
-51681363 781253
11846902 760019
70417706 447708
97846947 308016
71930324 538706
-92425218 15803
-89530890 830847
40610796 482840
-53935728 329170
50747481 933135
-93583350 996799
-84398602 970111
-2430916 681091
1038480 983079
28092828 709316
-99481874 928513
74294481 374288
16107253 672039
-43803799 927075
-55272043 644305
-25216452 118901
45608198 96173
-37907395 691140
14558241 846704
-77392161 710521
62367998 859910
-79612929 798209
38403354 482990
82202351 187556
95730452 832720
-45004954 724183
-87703794 697174
76574990 314140
-81347705 326789
88451406 832277
78205540 165020
-23646346 453491
-49461581 825603
-72178688 499870
58398917 520262
14504923 800949
-35261103 427662
55790690 417755
11846815 751947
-51030539 220212
77131712 699176
23222457 128797
-5593433 307470
-56894772 666989
-87593440 626904
-6041096 144223
-43907969 175935
-76290653 166742
-19783774 939883
-22720875 550985
95185473 810471
-61832689 995163
67039122 519622
-28445994 917111
44215856 184450
-35552639 393021
59025878 990308
16740411 159098
-99905645 365761
19388194 768656
-91092605 16817
-87990787 947098
4307043 855360
-55730468 365332
66200508 268246
24143046 883476
84002664 681301
25482752 590316
54539597 571062
-13477460 440174
85067275 238785
47166308 702577
-38728605 743693
27233828 713219
72426965 459580
-835322 253590
12195575 334293
20905699 401761
-14102805 728402
50281402 83542
-28109855 548121
-50775637 384707
-7554985 811708
54625775 174580
-70555115 865778
69138245 915289
28193091 581350
89191644 951079
32826548 68408
8860872 597739
-86318470 589192
-86167512 605119
-14508275 443779
54375153 112768
81749848 768202
-49839279 575624
47391496 422140
-924567 807235
-46997708 255370
-23446528 741339
79499953 486590
88390592 582435
-44921630 268801
24087764 327367
-23238494 473776
8406764 445930
-52470341 523019
25088732 738299
13779753 655451
94893102 613095
-19650223 934303
-26516371 561576
-84451827 123377
9103529 248903
51855551 201339
19364161 841606
-66741553 696634
73787767 824158
-25749212 698518
13406865 99443
-8493069 476813
-17417821 364365
4592447 155715
51075091 663729
-96499719 493954
20919307 594062
44725172 380076
22530352 394086
-43490831 200806
22808817 377419
19115736 510994
-94683935 861294
-26815235 881586
-61600512 722947
55409704 946957
-30264875 742376
-44054890 186924
-24661714 66730
-62273709 386284
16125894 353301
71820814 232816
-63295550 179095
-31275552 909940
-16555910 84326
-67684883 926179
-66208991 842920
49828737 562713
4691901 355213
-7080643 312879
25503014 822534
-12189129 115770
-10464511 54366
95920240 784784
46312005 967708
72781278 533082
97671555 945668
-77821679 918869
-16961568 156239
82994122 53756
10624421 623148
68506912 907479
-84818899 775577
55567924 866214
-78700335 171334
-90131060 168457
75262438 947367
-604230 890587
-97538314 186470
24996456 773165
-3962857 817662
83498189 12910
1483014 907803
-33803617 271567
-13252589 775997
76474457 684069
-20789350 237985
96002450 895491
56430531 7949
-80617784 339119
-31206668 212797
-30933159 253551
54295238 515532
68685810 627002
-3481735 485741
-8538075 662154
35507976 17068
-12377852 581173
-35781618 229857
-28003792 185006
25117565 788423
72809812 425926
91415175 432713
-52815620 765011
44157983 439031
-83212838 144624
17909218 54400
-50525566 295215
64265064 112945
96206958 617282
88507447 251310
36661388 879247
-33962717 665771
78864682 620040
96378375 77333
-82019098 561076
94708256 538186
-79618375 685498
29104132 202450
-43828537 552324
-93235830 317151
95260657 683999
59164523 111024
-17090695 245644
50205403 232713
-41337220 587198
-10851739 140530
-33162194 576254
-71500507 760137
70057067 858220
25618345 400399
14093013 114761
68020126 316188
-46894591 425942
36430156 671690
27549407 192079
-47088761 817288
5397487 497019
37375633 261569
34921583 754169
-80812799 944890
62252123 168432
-83287734 777125
-10908305 814528
64235204 578366
-56136313 106079
-7589911 579890
-61661339 25531
85431898 707130
-67819712 272294
85179686 29084
50053365 124850
2053532 639835
-73893296 99881
-20884276 101668
-37316346 881415
56000754 969749
8727210 193218
12546243 687930
58497568 143983
-71145935 748566
-66063552 868226
-35114975 917168
-76786322 83153
32646299 992520
47595605 742114
53714665 120620
-56812752 831610
-11612433 900640
-42538922 504147
97526510 316592
-97069831 319530
73426760 657897
55272215 905515
10344508 209978
-53224045 344200
11008345 350907
-52262520 622411
-83037793 679822
46495658 987903
4870287 797396
-90470430 194427
30300239 891420
70773585 768638
-49470767 773825
-1448297 856346
-44521145 215079
52387500 873333
42490410 388772
-67791108 321758
89073811 778217
25299927 626464
-11914984 316903
6711346 39877
-80395070 842020
75605800 976638
-82850363 49710
-98582330 283203
-45768048 77250
-76300169 760415
-80017787 349683
86920577 968618
8923393 281384
-66279448 831603
-41822621 851111
38739378 401408
-72527868 468750
-16804277 559745
-19601560 989546
35508098 316442
-43749475 198166
-38237539 159579
38174648 355337
-64118685 463980
7768885 868464
-6631022 120641
62765918 930932
95982376 397323
68553947 948115
-17085747 342654
73855624 753147
-93570339 379194
14084865 855326
86915215 466308
65216426 726954
96087634 247033
73335744 257953
-40553756 327012
-83842829 152011
-21833518 111750
-72884743 45957
-79424842 185567
-40956306 752430
-86556558 418045
43188528 965044
10983905 391803
85395042 721246
-99761851 984575
-4828897 516280
81256777 841690
26741008 695361
-75910311 590169
85150772 294690
-54791878 660378
79801649 200341
64397330 46129
-49245767 125752
-11496741 972749
-33671633 119990
50183567 741559
70027235 726466
3426015 962269
57363967 548762
64092474 229622
-29041297 61394
-76524328 495632
7770717 159018
-14763422 663075
-89204321 706519
42824964 899217
15839466 506826
35179710 62836
-3874338 560370
-18820340 410323
23506201 968692
15931536 446011
-92382349 257846
52543737 85763
-91140608 375427
5821361 549762
-59111012 276717
-78582473 797160
20330453 521644
-89923556 290327
22550226 453902
-19912058 817931
-62423890 715500
82806490 104767
3928010 550186
-1470998 578907
80673455 249856
-98083887 698017
-89657107 216132
42100661 641826
-36921780 692461
-57390022 931631
97684888 6225
83591387 217858
-42281732 853251
-1314016 616647
-17815082 679098
-58013128 952534
9095674 931912
82219050 153496
79322949 112923
49308353 630729
-21046497 867426
78871695 678094
-13918924 649566
-91604809 528584
94353663 582813
97179361 775328
-35384457 476076
1021423 620769
-15571110 615561
83446706 401285
10421785 146577
-36802455 955285
34789265 828189
-26244149 881591
-89310405 156970
54379115 696678
82364523 640533
59799589 984799
-67370009 691331
92348753 640007
46273335 117399
90197882 431539
78098295 728237
60314794 147317
-32695170 563324
11688625 918510
23566839 590061
74397319 329825
-49121922 372616
74795007 240523
-46303663 989835
-93035134 55197
-31906709 541058
71546848 380746
-35761469 937476
-57822946 273037
-66246441 830364
31919695 679523
-19949026 953697
50138970 72976
-38896740 471235
33524850 926228
78267716 403394
49317998 548261
-19982522 452181
-51932349 500538
91472872 998582
-62589026 922579
67917570 280634
29760229 445639
-92628211 967911
87696189 694747
34234314 979352
9043552 540658
-59306778 10238
-17318422 243174
-84638281 246319
36802450 783367
84803054 728694
92594043 226124
-70988528 887991
-70485223 768936
37355953 199480
-63002835 520244
-88852399 353950
-46042733 878229
-19158811 580066
-49302129 318351
61823825 620089
-1345494 170321
-23999714 313586
-73725612 506787
72096840 179332
-55752829 197295
90968783 748957
-33713454 742769
60080714 437379
-38516376 271654
72894303 529093
-53390700 903325
-90426795 960649
54102336 462766
-13181348 321108
-16262462 780511
-5208914 689282
76589194 342064
-65594334 28951
64126109 201121
-17049567 101942
-78017005 706194
64963083 392129
85511851 395738
-81594513 618265
-76599545 756359
26699054 947058
-86277240 771296
29643459 930683
53439354 22240
92911247 324532
29942564 960103
-78052454 300715
96549203 372577
65171797 902728
-86653978 213130
-79007150 76417
62672998 535740
-61314530 720022
-57804533 884903
15222352 772124
19054660 110077
-84752649 613762
-56789525 59394
-83924306 974818
34017768 157560
27363155 14933
64492249 325251
50172866 397320
-46998225 983554
21500544 962305
88830089 594689
-90992991 460755
-37274045 775182
-94673157 503512
-24541250 785640
-81577655 501898
16272909 178306
80299614 359903
-99573895 342064
69880459 155785
645351 506657
44654593 342845
6733394 436728
12802625 502314
-88909362 604827
-23535774 777599
7109335 760956
-33579128 270842
-80596942 660790
-66690330 241918
19060317 712002
68226158 71220
-66922355 787315
94645738 244393
-41314721 956107
77725995 933546
-16180840 496514
-65608419 347900
49425980 82055
96003068 57048
-31444599 109369
30764790 671510
-4277162 167028
-13456058 968667
78950370 188433
-71447365 370349
-11768634 331884
21587826 90771
93740303 72446
34391170 76456
-76389369 173124
-16090167 133847
38401464 685379
-91415225 941973
67116206 814433
50723494 457835
-10870354 522781
47733012 165010
83146742 706886
-48310725 588307
39532462 839510
-53532122 171151
85710272 405459
-31841159 419672
-31783063 967593
85700451 966525
29947086 861383
-45480422 955005
97675940 789996
-98340613 168565
26973299 942791
84773809 358281
55388144 941783
-69836863 810875
73426725 913809
-365003 601285
46856134 925907
-15398811 11925
-58901772 649749
-55899181 95685
-21717835 460223
-80062694 104345
-91090084 605392
-53083618 499365
47827059 18959
-35285842 821546
-21676398 946855
13865449 554360
91913869 482347
-47580027 611286
34802324 673805
-74038091 986278
80379311 222875
-38996279 518045
27421658 53433
-65710647 76635
-10887361 825567
-93549641 247676
22443879 947795
7253571 590676
21051168 701809
-13318050 46955
-75338077 714314
31427729 500194
-60847995 335542
29676160 429294
68405266 326131
94557022 889494
-89143088 721564
-56522338 287344
-16627176 697205
36431381 483786
5582173 21793
90634954 683970
-48900697 63564
-62364763 382096
-73890472 717822
-68700449 207189
-93829259 89885
79679878 733484
-60048017 917580
50619744 922022
58786734 160745
-76465380 373261
88266235 618744
-49978159 822670
-8475481 597953
-44991765 403698
-97915010 153568
47181254 713499
15440282 61220
-32436399 307451
14642991 239923
-15667760 532207
-68409829 834818
45415049 614312
-34927721 987369
45285827 150678
-66940977 535943
-98311680 339702
28013055 239902
19612564 11849
-13595897 84676
9630527 649257
71037386 788567
41233729 779431
97858264 615004
-22385634 638902
-5773261 473673
-86151910 216569
36692484 518300
-80737266 79797
-7738206 8286
-77034786 514784
-57483256 875379
26534428 56273
-76210011 984960
-49915617 261917
97954612 479440
95898524 665792
86065858 346085
-15812163 801878
-11806364 44635
69857145 156606
-38946438 410934
-72578186 681193
-88995477 762428
84829008 72599
-75141228 786925
99199043 921771
74577247 181875
-48017440 181255
61625756 391121
98330284 251006
72635508 179209
-87594759 303814
-57172707 798563
21407771 773721
98769888 122641
-3407914 280828
-99365116 419731
-32374128 777236
-84670358 95357
-23817769 228934
-89972543 177042
-60653965 116712
47067901 439069
-14573089 311488
55388478 501195
74419386 34364
97811454 699499
-7449822 866052
47964493 210788
-87061281 113814
-45784445 415661
20685132 157483
-41505492 887530
-43612691 492300
