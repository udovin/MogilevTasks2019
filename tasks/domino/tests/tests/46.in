421
-80667373 37161
-82372439 8805
-84079449 16151
-80880363 70089
-85507263 22383
-95231040 44055
-95762524 95196
-67067416 7308
-78499927 26279
-86418510 116352
-87204254 31710
-87454147 9838
-70003703 6506
-93150286 9368
-86170304 46641
-83615518 33857
-91397348 57233
-81540884 19740
-88367316 137851
-98032285 64346
-92692520 881
-92108773 13231
-68487209 15017
-92872614 8296
-89602427 977
-67266834 6052
-93968989 29534
-84862301 79482
-67309696 82417
-66321501 83928
-89262679 28314
-90738833 65173
-71938515 14607
-97573383 25185
-88617532 26877
-71095930 33715
-91734212 43312
-71684442 88320
-69870035 70606
-78320378 81131
-91537717 99686
-95009905 21005
-92372853 12608
-85381839 4510057
-70091060 70387231
-74868900 10272
-71463512 13136708
-93199740 106505
-93406330 62424173
-82644189 32919510
-94619792 21977
-78574600 50618
-96587188 65379
-96156507 66264
-77654203 62025
-75298520 91117
-79011606 54738
-72144277 59061
-68357539 27106
-98614202 57933
-82244697 14432
-76785342 38237
-94862458 32395
-95099703 54786406
-83127523 39118
-87925822 1937
-90502526 7584
-68594636 129096
-86090331 9264
-70522385 42891
-79941283 18813
-69752495 108455
-65750919 55111
-92668427 1174
-85328083 2012
-72994584 36012
-81449285 61469954
-88253609 43727
-98466800 19653
-91668933 40946
-67436608 1664
-73400482 315
-64241176 36561
-87360434 76460
-81964408 16546
-76475579 20077
-96952812 24030
-68929339 91072
-64131264 5284
-73835582 114181
-86803907 14056
-92239724 127695
-71216524 27290
-71823929 57858
-67823533 6869
-79227947 102211
-73709983 6966
-84389647 35236
-68505392 17320
-81278983 64802
-89441686 1829
-67447780 28121
-84827206 13003
-66654869 75565861
-94771952 44533
-67507245 49424
-83284153 12489
-70878723 17268
-96423813 21471
-88194813 52852
-79676574 86903
-87043649 4924
-70952831 81610
-72592607 61752
-95330999 93773
-82242957 1269
-88660589 6962
-66045499 112856
-85717331 32133
-85193661 46059
-83346737 106600
-82432614 1892
-80429842 4461
-74648362 13506
-64049117 27840
-85181788 2358
-73108007 140126
-89135102 1113
-96670411 4568
-75397496 60409
-94433651 37388
-97225324 56443
-88894971 102045
-67693093 12053
-72729401 117124
-85095741 9587
-97874893 33795
-81777435 33803
-71159699 16212
-91251875 8889
-78262472 30194
-77042892 1852
-70465463 798
-85977315 78276133
-68725991 3708
-79795738 24330
-83728993 41531
-81105304 13308
-98211240 38600
-90100246 121497
-68795112 4403621
-77431424 17786
-94966650 9954
-78008257 34572
-82434522 23973
-98753928 23755
-94306305 32634
-95573116 55344
-97344026 1464
-82614908 14887
-91114040 10275
-71203444 11188
-96052732 94458
-69225773 50476
-71334767 59527
-80862229 17050
-93524693 25887
-77117087 4369851
-70375289 6486
-96314509 2278
-81825647 8047
-75636819 25326
-77338619 46438
-86798856 1219
-97397039 28307
-87879694 37403
-91874036 19995
-86631621 37697
-69498159 16136
-74858202 5069
-78836407 1908
-94526184 41051
-83858260 22065
-87595084 15274
-92743378 79513
-65639973 68921
-76755597 26927
-82461410 65427
-98309354 74406
-70086256 946
-98789771 85304
-88088677 93372
-85349752 1857
-93793119 6218
-75500609 83512
-97817793 11247
-68123998 35700
-88740025 11425
-69060738 7193
-64698708 66715
-69344967 55118
-93850220 86638
-83098184 13648
-90958755 77248
-67082737 10410
-81372726 70450
-82900138 57697
-82207718 5929
-68278295 3394352
-80070824 57577
-83000440 91275
-73250284 18185
-78144748 29228
-94575096 6837
-89337560 84748440
-73309064 30617
-70656864 32553
-70943460 2549
-70291434 22842
-67997390 49424567
-65340183 33062
-77875006 19875
-87634585 116052
-90251401 22192
-81682345 1385
-92497766 21538
-63673493 135645
-98100650 50582
-80279138 3145
-68741861 16317
-63811881 31621
-80283957 9773
-73627551 58820
-64772907 215
-65880247 43077
-69655813 22517
-95723544 15707
-81872154 65395
-85655633 28413
-83990255 47725
-66508343 867
-74242796 41515
-80401379 27308
-93009963 64257
-65948326 20051
-81689802 42062
-70420479 36616
-75282833 10092
-76612218 97816
-89634789 135799
-84979798 17652
-93646623 21686063
-69164571 25134
-88513546 22112
-84661316 34940
-74407876 26511
-92602025 35577
-91127173 109046
-80764615 50677
-72301322 32624
-76904087 9625
-70226744 25400
-78840045 121401
-64924555 92196
-81133367 63846
-86712289 53067
-73568095 8209
-91296123 37137190
-80706894 15725
-91037457 47986
-72377575 1411
-74204199 18956
-74723533 127819
-97688250 5757
-65227537 78399
-73533703 32620
-76319127 66101
-77905696 74694
-82073567 104327
-64872765 44689
-69083760 9467
-75742227 3574
-81531262 4834
-80999062 97359
-89951642 12787
-74320195 37371
-77524313 84487
-85856921 19492
-66795178 36538
-98520500 83925
-92675337 16625
-80450135 26852
-79522079 68115
-76431014 42994
-89485503 23590
-63865761 61315
-90580366 10321
-84812499 14693
-80731097 4515
-82402418 28954
-71593790 54867
-73471923 12154
-72395709 59313
-72357647 15575
-77317124 12634
-96675727 34071
-83494130 11168
-76922850 50002
-74902139 80398
-75755466 79209
-96797251 79994
-80181308 20408
-70053822 2881
-85905740 27243
-79865139 57001
-73344781 2012
-90642681 16592
-73282830 1979
-95919367 58415
-65053930 1010
-89308330 9730
-85772772 31439
-97506399 33774
-86932007 84224
-64433753 84901
-92020147 70645
-89784873 27408983
-95439794 14989
-78416522 8215145
-79454916 54142
-69551868 57792
-89121583 3779
-66935656 110181
-63942096 16710
-87991464 96970
-69101116 8546
-83242769 6881
-79362327 80721
-75058722 42153
-71956674 43006
-83776119 59005
-79339849 19073
-93309160 1054
-67114987 103518
-79636048 17262
-68169160 81478
-97979964 1405
-66881305 50282
-74081752 35805
-66439706 54201
-65585126 44814
-75997583 85310
-67864506 127436
-94352730 27172
-65382862 44554
-80530219 46700
-78971899 15418
-89137166 67578162
-70756212 23156
-89448372 27763
-65082451 82084
-87785645 7139
-75845583 8329
-64323411 28890
-66190339 57428
-67600522 90637
-90381631 1925
-97102980 115435
-86349153 19727
-95298736 10460
-94170443 80108
-82006799 51376
-82804553 68144
-89936189 11289
-78722622 113613
-97780133 24856
-76288869 29413
-84525897 113820
-71462077 334
-73801985 18248
-73988215 6746
-85464155 3926
-82777769 7952
-86902381 24608
-88760704 36989
-74291436 22130
-96301065 3139
-74505279 79740
-76148374 13166
-82183582 2971
-76516772 22747
-95037335 19263
-90869423 80435639
-72035344 1473637
-90350747 12072
-96479966 91827348
-78527868 28059
-73966783 12312
-98178677 30452
-86217963 92231
-75135803 18439
-75636428 125
-79135679 35312
-88310756 54244
-75662272 10475
-93324214 18680
-94038850 112692
-88573453 27103
-77718092 2985513
-72458264 128043
-76166802 109801
-94929883 30319
-86541457 42447
-72873445 29879
-64572277 80072
-87050074 36982
-77256729 51684
-65475343 47057520
-89001027 62455
-84133615 100266
-84285637 44475
