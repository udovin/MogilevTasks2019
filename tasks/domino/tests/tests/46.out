1 1 1 2 1 1 2 1 1 2 1 1 1 1 1 1 1 2 3 2 1 1 1 1 1 1 1 3 2 1 1 1 1 1 1 1 1 1 1 2 1 1 1 266 73 1 91 2 361 235 1 3 1 1 1 2 2 1 1 1 2 2 1 381 2 1 1 3 1 1 1 2 1 1 1 1 214 1 1 1 1 1 1 1 1 1 1 1 1 2 2 1 2 1 1 2 1 1 1 1 1 2 1 31 1 1 1 1 1 1 2 1 3 1 2 1 1 2 1 2 3 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 274 1 1 1 1 2 1 57 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 56 1 1 1 2 2 1 1 1 1 1 1 1 1 1 1 1 4 2 1 4 1 1 2 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 50 1 1 1 1 1 316 2 1 1 1 47 1 1 2 1 1 1 1 1 1 2 1 2 1 1 1 1 1 2 1 1 1 2 1 1 1 2 1 1 2 2 1 363 1 1 1 1 1 3 2 1 1 1 3 2 2 1 1 339 1 1 1 1 2 1 1 1 2 2 3 1 1 1 1 1 1 2 1 1 1 2 2 2 2 1 1 2 1 1 1 1 1 1 3 1 1 2 1 2 3 2 1 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 322 1 175 1 2 1 2 1 2 1 1 2 1 2 2 1 1 3 1 2 1 1 1 1 1 1 2 1 2 1 1 313 1 2 2 1 1 1 1 1 1 1 1 1 1 2 2 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 333 97 1 395 2 1 1 2 1 1 1 1 1 2 2 1 63 4 2 1 1 1 1 2 1 20 1 2 1 
